.ci/scripts/docker/upload-gcr.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/sh -eux
+
+# this command is a little convoluted to avoid leaking of the secret;
+# it is unclear why the built-in Jenkins mechanism doesn't work out of the box
+echo "Authenticating with gcr.io and pushing image."
+set +x ; echo ${DOCKER_GCR} | docker login -u _json_key --password-stdin https://gcr.io ; set -x
+
+docker push ${IMAGE}:${TAG}
+
+
```

Comment:
```
[zell docker/ ns:zell-job-activiation]$ shellcheck upload-gcr.sh 

In upload-gcr.sh line 6:
set +x ; echo ${DOCKER_GCR} | docker login -u _json_key --password-stdin https://gcr.io ; set -x
              ^-----------^ SC2086: Double quote to prevent globbing and word splitting.

Did you mean: 
set +x ; echo "${DOCKER_GCR}" | docker login -u _json_key --password-stdin https://gcr.io ; set -x


In upload-gcr.sh line 8:
docker push ${IMAGE}:${TAG}
            ^------^ SC2086: Double quote to prevent globbing and word splitting.
                     ^----^ SC2086: Double quote to prevent globbing and word splitting.

Did you mean: 
docker push "${IMAGE}":"${TAG}"

For more information:
  https://www.shellcheck.net/wiki/SC2086 -- Double quote to prevent globbing ...

```